frontend/test/tickets/tickets.store.spec.ts

```diff
@@ -38,6 +39,21 @@ describe('Tickets: store', () => {
 		expect(modelTicketsFromState[0]).toEqual(ticket);
 	});
 
+	it('should update the model tickets', () => {
```

Comment:
Could you test the upsert method for its `insert` functionality as well (without the given ticket existing in the store)?